### DISCUSSION TITLE
Question- Time issue?

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7775] [QA - tech support + general] [answered]
for me its 10:29 am and i wanna know, what time do i set it for it to be morning in MAS? rn it is not morning in mas

settings:

![Screen Shot 2021-09-04 at 10 23 53 AM](https://user-images.githubusercontent.com/82217573/132099805-a813640e-e0a4-4731-9a41-cec9f7d4f085.png)

**Comment by Booplicate** [comment]
[Sunrise](https://en.wikipedia.org/wiki/Sunrise) is the time when morning starts, [Sunset](https://en.wikipedia.org/wiki/Sunset) is when night starts.

**Comment by ghost** [comment]
what time do i put them as if i want it to show as morning then?

**Comment by Ulti70895** [comment, marked as answer]
If you want morning at 10:30 then set Sunrise at 10:30.
